### DISCUSSION TITLE
Event on autoplay stop

### DISCUSSION THREAD

**Discussion by hugo-ym** [igordanchenko/yet-another-react-lightbox#198] [Q&A] [answered]
Hello. I want an autoplay on the slideshow, that is OK. But when the user stop the autoplay, i want to register the state on local storage for the next time. How can i can catch an event when click on stop button ?

**Comment by igordanchenko** [comment, marked as answer]
You can use `on.slideshowStop` callback - https://yet-another-react-lightbox.com/plugins/slideshow

**Reply by hugo-ym** [reply]
That's the page i use but without success. Lightbox is OK, autoplay is OK but impossible to have an event when click on stop or start manually on buttons. Is something wrong in my code wrong:
```
<Lightbox
        render={{
          iconNext: () => <MyNextIcon />,
          iconPrev: () => <MyPreviousIcon />,
        }}
        on={{
          slideshowStart: () => console.log('aaa'),
          slideshowStop: () => console.log('bbb'),
        }}
        slides={data}
        captions={{ descriptionTextAlign: "center", titleTextAlign: "center" }}
        slideshow={{ autoplay, delay }}
        plugins={[Inline, Slideshow, Video, Thumbnails, Captions, Fullscreen]}
      />
```

**Reply by igordanchenko** [reply]
I do not see anything wrong with your code. `slideshowStart` / `slideshowStop` are properly called when I click the 'play' / 'pause' slideshow button. Please consider sharing a complete minimal reproduction on CodeSandbox if you require further assistance.

**Reply by hugo-ym** [reply]
It's a part of a big code, so not easy to reproduce. If you say that my code seems ok, i think i must have another issue with another piece of code. I will try to remove some parts and test. Thank for response.
